speech/google/cloud/speech/client.py

```diff
@@ -299,6 +298,74 @@ def sync_recognize(self, sample, language_code=None,
                                   profanity_filter, speech_context)
 
 
+class StreamingSpeechResult(object):
+    """Streaming speech result representation.
+
+    :type alternatives: list
+    :param alternatives: List of protobuf speech alternatives.
+
+    :type is_final: bool
+    :param is_final: Boolean indicator of results finality.
+
+    :type stability: float
+    :param stability: 0.0-1.0 stability score for the results returned.
+
+    :rtype: :class:`~google.cloud.speech.client.StreamingSpeechResult`
+    :returns: Instance of ``StreamingSpeechResult``.
+    """
+    def __init__(self, alternatives, is_final=False, stability=0.0):
+        self._alternatives = [Alternative.from_pb(alternative)
+                              for alternative in alternatives]
+        self._is_final = is_final
+        self._stability = stability
+
+    @classmethod
+    def from_pb(cls, response):
+        """Factory: construct instance of ``StreamingSpeechResult``.
+
+        :type response: :class:`~google.cloud.grpc.speech.v1beta1\
+                               .cloud_speech_pb2.StreamingRecognizeResult`
+        :param response: Istance of ``StreamingRecognizeResult`` protobuf.
+
+        :rtype: :class:`~google.cloud.speech.client.StreamingSpeechResult`
```

Comment:
Since you are in the same module as the referenced class (obviously, since this classmethod is part of the class) you can refer to it as

```
:class:`StreamingSpeechResult`
```